Konata.Core/Events/Model/OfflineFileDownloadEvent.cs

```diff
@@ -0,0 +1,37 @@
+namespace Konata.Core.Events.Model;
+
+public class OfflineFileDownloadEvent : ProtocolEvent
+{
+    /// <summary>
+    /// The bot url
```

Comment:
bot url?

Reply:
Generated by copilot and sleepy head -。-